rioxarray/raster_array.py

```diff
@@ -774,9 +777,12 @@ def to_raster(
             A dictionary of tags to write to the raster.
         windowed: bool, optional
             If True, it will write using the windows of the output raster.
-            This only works if the output raster is tiled. As such, if you
-            set this to True, the output raster will be tiled.
+            This is useful for loading data in chunks when writing. Does not
+            do anything when writing with dask.
```

Comment:
Does this still "force" tiled output files?

Reply:
No. You need to use `tiled=True` for that.

Reply:
Ok so this is a change in how windowed operates?

Reply:
It was changed back [here](https://github.com/corteva/rioxarray/pull/134) and I missed the docstrings.

Reply:
Ooohhh. That explains it. So if I do `to_raster(..., tiled=True, blockxsize=512, blockysize=512)` it will store the dask arrays with that internal geotiff tile size, right (and not the dask chunk size)?

Reply:
That's correct. Currently not automatically tiling according to the dask chunk size at the moment. So it needs to be done by the user.

Reply:
Good. Not sure auto-tiling based on chunk size would be a good idea anyway. Just checking.